presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStats.java

```diff
@@ -41,6 +42,7 @@
 
     private final Map<String, OperatorInputStats> operatorInputStats;
     private final Map<String, OperatorHashCollisionsStats> operatorHashCollisionsStats;
+    private final Optional<WindowNodeStats> windowNodeStats;
```

Comment:
Make sense but regardless what you agree on, I claim such a change should be a separate PR (to apply new schema to all stats at once).